src/main/java/com/gruzini/tennistico/scheduler/MatchStatusChangePublisher.java

```diff
@@ -17,33 +18,34 @@ public MatchStatusChangePublisher(ApplicationEventPublisher applicationEventPubl
         this.applicationEventPublisher = applicationEventPublisher;
     }
 
-    @Scheduled(cron = "0 0/1 * * * *")
+    //TODO: discuss scheduling for this event, for now used cron.upcoming
+    @Scheduled(cron = "${scheduling.cron.upcoming}")
     public void publishHeldGameEvent() {
         applicationEventPublisher.publishEvent(new HeldMatchEvent(this));
     }
 
-    @Scheduled(cron = "0 0/15 * * * *")
+    @Scheduled(cron = "${scheduling.cron.upcoming}")
     public void changeStatusFromJoinRequestToBusted() {
         final ChangeMatchStatusByStartingDateTimeEvent event =
                 new ChangeMatchStatusByStartingDateTimeEvent(this, MatchStatus.JOIN_REQUEST, MatchStatus.BUSTED);
         applicationEventPublisher.publishEvent(event);
     }
 
-    @Scheduled(cron = "0 0/15 * * * *")
+    @Scheduled(cron = "${scheduling.cron.upcoming}")
     public void changeStatusFromHostedToBusted() {
         final ChangeMatchStatusByStartingDateTimeEvent event =
                 new ChangeMatchStatusByStartingDateTimeEvent(this, MatchStatus.HOSTED, MatchStatus.BUSTED);
         applicationEventPublisher.publishEvent(event);
     }
 
-    @Scheduled(cron = "0 0 * * * *")
+    @Scheduled(cron = "${scheduling.cron.pending}")
```

Comment:
Spoczko, ale dla lepszego trackowania zmian takie rzeczy można by robić na osobnym PR ;)

Reply:
Prawda, nawet przez chwilę miałem na osobnym branchu, ale były jakieś problemy ze znalezieniem zmian w pomie (w sensie nie znajdowało) co sprawiłoby, że po zmerdżowaniu osobnego brancha pewnie by nie działało. Aczkolwiek jak się potem zastanowiłem, to nawet wiem skąd to wynikło, tak więc mb, przerzucę to do osobnego brancha.

Reply:
Nvm, wolę jednak nie tykać tego, jak czytałem o tym jak discardować zmiany z PR to pojawiały się wzmianki o Force Pushach, więc chyba wolę to zostawić w ten sposób i nie płakać, że coś się zrypało. Chyba, że wiesz jak to zrobić porządnie.